src/appleseed.studio/mainwindow/renderingsettingswindow.cpp

```diff
@@ -510,6 +515,7 @@ namespace
             m_image_plane_sampler_combo->setToolTip(m_params_metadata.get_path("sampling_mode.help"));
             m_image_plane_sampler_combo->addItem("Uniform", "uniform");
             m_image_plane_sampler_combo->addItem("Adaptive", "adaptive");
+            m_image_plane_sampler_combo->addItem("Adaptive Tile", "tile-adaptive");
```

Comment:
Maybe `adaptive_tile` instead of `tile-adaptive`?

Reply:
Yes :)